examples/polynomial/evaluation/ensemble.yml

```diff
@@ -8,4 +8,5 @@ input:
 forward_model:
   driver: local
   stages:
-    - evaluate_polynomial
+    - name: evaluate_polynomial
```

Comment:
The stage itself was not intended to be defined within the `forward_model`, but inside the `stage.yml`. If you merge it like this first I suggest you make an issue to move this logic out into the `stage.yml` again. It is good for discussion if we at least semi-adhere to current agreement with the owls such that we can continue joint discussions...

Reply:
The reason for this implementation is that the `stage.yml` has not yet been implemented, and I wanted a somewhat limited scope to this PR, can make an issue or implement that now.

Reply:
That is up to you! It is perfectly fine to leave it in an issue also, but then we should at least create the issue :)